dspace-xmlui/src/main/java/org/dspace/app/xmlui/wing/element/Field.java

```diff
@@ -488,6 +489,28 @@ protected void removeValueOfType(String removeType)
     }
 
     /**
+     * build the languages select for metadata values
+     * 
+     * @param options
+     * @throws WingException
+     */
+	public void setLanguagesList(List<String> options) throws WingException 
+	{
+	 
+		List<Option> langOptions = new ArrayList<Option>();
+		 
+		for (int i = 0; i < options.size(); i += 2)
+		{
+			String display = options.get(i);
+			String value   = options.get(i+1);
```

Comment:
it will, these options are defined in a <value-pair> from the input-forms.xml, and according to the <pair> definition in the input-form.dtd it will always have exactly two children